src/main/kotlin/frc/robot/subsystems/shooter/flywheel/Flywheel.kt

```diff
@@ -40,9 +40,21 @@ class Flywheel : SubsystemBase(), SysIdable {
                 velocitySetpoint = velocity
                 mainMotor.setControl(velocityTorque.withVelocity(velocity))
             }
-            .withName("Flywheel/setVelocity")
+            .withName("$name/setVelocity")
+
+    fun setVelocity(velocity: Supplier<AngularVelocity>): Command =
+        run {
+                velocitySetpoint = velocity.get()
+                mainMotor.setControl(
+                    velocityTorque.withVelocity(velocity.get())
+                )
+            }
+            .withName("$name/setVelocity")
```

Comment:
I'd use Kotlin's syntax here:
```suggestion
    fun setVelocity(velocity: () -> AngularVelocity): Command =
        run {
            val velocityValue = velocity.invoke()
            velocitySetpoint = velocityValue
            mainMotor.setControl(
                velocityTorque.withVelocity(velocityValue)
            )
        }
            .withName("$name/setVelocity")
```
At least, extract to a variable -- otherwise you risk having a specific value in line 47, and another one in line 50.
Don't forget to remove the supplier import.

Reply:
ok e23b8064a6aebe5287eee00c0c79b64f2fba607c